.changes/aliasoutputbuilder.md

```diff
@@ -0,0 +1,6 @@
+
+---
+"nodejs-binding": patch
+---
+
+IAliasOutputBuilderOptions::stateMetadata is now a HexEncodedString instead of Uint8Array.
```

Comment:
Also `IAliasOutputBuilderOptions::aliasId` and `INftOutputBuilderOptions`

Reply:
They're still a string, it's just named so it's easier to understand what's needed

Reply:
Ah yeah I see 👍🏻